index.md

```diff
@@ -0,0 +1,29 @@
+## Malagasy singers:
```

Comment:
This should be a title heading.
Please also make it read: "Malagasy artists" as not all artists are singers, right?